modules/demo-magento/cypress/integration/checkout.js

```diff
@@ -61,4 +61,32 @@ describe('Checkout', () => {
 
     thankYouPage.isVisible()
   })
+
+  it('add valid coupon', () => {
+    product.visitRandom()
+    product.addToCart()
+    product.continueToCheckout()
+
+    checkout.getAddCoupon().click()
```

Comment:
Can't these bunch of steps be put together to one method in a page object called applyCoupon() or something akin to addToCart() in product page object?